browser/js/components/Comment/renderComment.js

```diff
@@ -47,14 +47,18 @@ export default class RenderComment extends Component {
         open: false
       }
     }
-    this.updateContents = this.updateContents.bind(this);
+    // this.updateContents = this.updateContents.bind(this);
     this.toggleDialog = this.toggleDialog.bind(this);
     this.renderDialog = this.renderDialog.bind(this);
     this.renderComment = this.renderComment.bind(this);
     this.handleSubmitDialog = this.handleSubmitDialog.bind(this);
     this.renderMarkdown = this.renderMarkdown.bind(this);
     this.renderScore = this.renderScore.bind(this);
-    this.removeItem = this.removeItem.bind(this);
+    this.removeItem = this.props.removeItem;
+    this.updateContents = this.props.updateContents;
+    this.renderDialogHandler = this.renderDialogHandler.bind(this);
+    this.tagMethods = this.props.tagMethods;
+    this.handleSlider = this.handleSlider.bind(this);
```

Comment:
You could use es6 to reduce the lines taken. E.g.

const { removeItem, updateContents, rengerDialogHandler, tagMethods, handleSlider } = this.props 😄